### PR TITLE
Fix vector GitHub actions

### PR DESCRIPTION
**Pull request by smohiudd**
Fix github actions for dev deployment. Include missing secrets declarations.

### CODE REVIEW

File: .github/workflows/deploy_dev.yml [hunk 1]
```diff
@@ -16,5 +16,4 @@ jobs:
       external_role_arn: ${{ secrets.external_role_arn }}
       vector_secret_name: ${{ secrets.vector_secret_name }}
       vector_vpc_id: ${{ secrets.vector_vpc_id }}
-      vector_security_group: ${{ secrets.vector_security_group }}
-
+      vector_security_group: ${{ secrets.vector_security_group }}
```
Comment (slesaad) [inline .github/workflows/deploy_dev.yml:19]: new line

Comment (slesaad) [reply .github/workflows/deploy_dev.yml:19]: can you also add these changes to `deloy_staging.yml` ?

Comment (slesaad) [reply .github/workflows/deploy_dev.yml:19]: although we maynot have values for vector staging yet

Comment (smohiudd) [reply .github/workflows/deploy_dev.yml:19]: yeah that's why I didn't include it just yet. But could include it for completeness

**Review by slesaad — APPROVED** [review]
LGTM

**Pull request merged**